components/automate-cli/cmd/chef-automate/ha_cert_show.go

```diff
@@ -0,0 +1,414 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/chef/automate/components/automate-cli/pkg/status"
+	"github.com/chef/automate/components/automate-deployment/pkg/cli"
+	"github.com/spf13/cobra"
+)
+
+const (
+	CONST_AUTOMATE    = "automate"
+	CONST_CHEF_SERVER = "chef-server"
+	CONST_POSTGRESQL  = "postgresql"
+	CONST_OPENSEARCH  = "opensearch"
+)
+
+type certShowFlags struct {
+	node string
+}
+
+type certShowImpl struct {
+	flags     certShowFlags
+	nodeUtils NodeOpUtils
+	sshUtil   SSHUtil
+	writer    *cli.Writer
+}
+
+type certShowCertificates struct {
+	//Automate Cets
+	AutomateRootCert  string
+	AutomateCertsByIP []CertByIP
+
+	//ChefServer Cets
+	ChefServerCertsByIP []CertByIP
+
+	//Postgresql Cets
+	PostgresqlRootCert  string
+	PostgresqlCertsByIP []CertByIP
+
+	//Opensearch Cets
+	OpensearchRootCert  string
+	OpensearchAdminKey  string
+	OpensearchAdminCert string
+	OpensearchCertsByIP []CertByIP
+}
+
+func init() {
+	flagsObj := certShowFlags{}
+
+	certCmd := &cobra.Command{
+		Use:   "cert COMMAND",
+		Short: "Chef Automate Certificate Management",
+		Long:  "Chef Automate certificate management, this command should always be executed from AutomateHA Bastion Node.",
+	}
+
+	certShowCmd := &cobra.Command{
+		Use:   "show",
+		Short: "Chef Automate Certificates Show",
+		Long:  "Chef Automate CLI command to show certificates, this command should always be executed from AutomateHA Bastion Node",
+		RunE:  certShowCmdFunc(&flagsObj),
+	}
+
+	certShowAutomateCmd := &cobra.Command{
+		Use:   "automate",
```

Comment:
we are following some patterns to refer to the services as flags, so I recommend using the same flags.
https://deploy-preview-7604--chef-automate.netlify.app/automate/cli_chef_automate/#chef-automate-cert-rotate